backend/.eslintrc.json

```diff
@@ -23,6 +23,7 @@
           }
```

Comment:
Also think we should add .prettierrc.js, this is mostly for the spacing, but does help with a few other things, similar to the front end.

Reply:
this is a good point. i will add it now, so that this is only one bandaid to rip off.

Reply:
Yes, agree 100%